contracts/Swap/SwapRouter.sol

```diff
@@ -16,7 +16,7 @@ import "./interfaces/InterfaceComptroller.sol";
  * @author 0xlucian
  */
 
-contract SwapRouter is Ownable2StepUpgradeable, RouterHelper, IPancakeSwapV2Router {
+contract SwapRouter is Ownable, RouterHelper, IPancakeSwapV2Router {
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
     address public comptrollerAddress;
```

Comment:
`comptrollerAddress` can't be modified, maybe we could make it immutable for gas savings?

Reply:
Fixed, d066bf93502d88423e3aafe3106bda584cef3103